test/_testServer.js

```diff
@@ -3,6 +3,13 @@
 var jsonApiTestServer = require("jsonapi-server/example/server.js");
 if (!jsonApiTestServer.start) return;
 
+jsonApiTestServer.getExpressServer().use(function (req, res, next) {
+  res.header("Access-Control-Allow-Origin", "*");
+  res.header("Access-Control-Allow-Methods", "POST, GET, PUT, DELETE, OPTIONS");
+  res.header("Access-Control-Allow-Headers", "Content-Type");
+  next();
+});
```

Comment:
Even with all these headers, iOS on SauceLabs throws CORS errors :'( As does Firefox on the Travis instance...